pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: dartis
-version: 0.4.0
+version: 0.5.0
 author: Juan Mellado <jcmellado@gmail.com>
 homepage: https://github.com/jcmellado/dartis
 description: >-
   Redis client featuring type-safe commands, pipelining,
   fire and forget, publish/subscribe, monitor mode, inline
   commands, transactions, Lua scripting, modules, and much more.
-documentation:
 
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: ">=2.3.0 <3.0.0"
```

Comment:
Why not upgrade to Dart 2.12 for sound null safety?

Reply:
This PR is meant as preparation for a null safety migration. Tidying up usages of lists and introducing `pedantic` are initial steps towards a migratable codebase.